frontend/components/MockComponent/ActionButton.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { View, Text, StyleSheet } from 'react-native';
+import { useTheme, FAB } from 'react-native-paper';
+
+function ActionButton(props) {
+    const {colors} = useTheme();
+    return(
+        <FAB
+            style = {styles.fab}
+            icon = 'plus'
+            onPress = {props.onPress}
+            color = 'black'
+        />
+    );
+}
+
+const styles = StyleSheet.create({
+    fab: {
+      position: 'absolute',
+      margin: 16,
+      right: 0,
+      top: 700
```

Comment:
Let's use a different way of positioning this button towards the bottom right of the screen. What if my screen is shorter than 700px tall? (It actually is in my case and I can't see the button). I think using `bottom: 0` will do the job.